cli/gardener_ci/_cfg_mgmt.py

```diff
@@ -135,3 +141,63 @@ def rotate(
         logger.info('Rotation successful')
     else:
         logger.warning('Config element was not rotated.')
+
+
+def create_or_update_tickets(
+    cfg_dir: str,
+    issue_repo: str, # must be of format hostname/owner/repo
+    delivery_endpoints_cfg_name: str,
+    overwrite_assignee: [str]=[],
+    cfg_element_type: str = None,
+    cfg_element_name: str = None,
+):
+    def is_target(
```

Comment:
this name is greatly misleading. from its name, I would have expected the function to determine whether or not the passed arguments _are_ a target. what the function actually does is comparing whether or not the given cfg_element has name and type(-name) of passed-in name and type(-name) arguments!